stories/covid-vax/covid_vax.ipynb

```diff
@@ -0,0 +1,1527 @@
+{
```

Comment:
<p>Two narratives here. Break the second paragraph in two, one talking about what the graph is going to show, and another about the function to build the figure is doing.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alan-turing-institute/TuringDataStories/pull/138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>